src/app/components-small/plugin-filter-node/plugin-filter-node.component.ts

```diff
@@ -36,47 +39,47 @@ export class PluginFilterNodeComponent implements OnInit {
             console.warn("No filter object provided to plugin filter node component");
             return;
         }
-        const isLeaf = (filter.name != null) || filter.tag || filter.version;
+        // Should filters have multiple attributes at some point, this must be changed
+        const type = Object.keys(filter)[0];
```

Comment:
If you want to, you could add an active warning to the code:
```suggestion
        const filterKeys = Object.keys(filter)
        if (filterKeys.length != 1) {
            console.error("Filters with more than one attribute are not supported!");
        }
        const type = filterKeys[0];
```